tests/helpers/mobiledoc.js

```diff
@@ -0,0 +1,26 @@
+import PostNodeBuilder from 'content-kit-editor/models/post-node-builder';
+import MobiledocRenderer from 'content-kit-editor/renderers/mobiledoc';
+
+/*
+ * makeMD({post, section, marker, markup} =>
+     post([
+       section('P', [
+         marker('some text', [markup('B')])
+       ])
+     })
+ */
+function makeMD(treeFn) {
```

Comment:
Can this be `makeMobiledoc`?

Additionally, those docs do not look like valid syntax.